runtime/moonbase/src/lib.rs

```diff
@@ -520,6 +531,91 @@ impl pallet_author_mapping::Config for Runtime {
 	type AuthorId = NimbusId;
 }
 
+parameter_types! {
+	// One storage item; key size 32, value size 8; .
+	pub const ProxyDepositBase: Balance = currency::deposit(1, 8);
+	// Additional storage item size of 33 bytes.
+	pub const ProxyDepositFactor: Balance = currency::deposit(0, 33);
+	pub const MaxProxies: u16 = 32;
+	pub const AnnouncementDepositBase: Balance = currency::deposit(1, 8);
+	pub const AnnouncementDepositFactor: Balance = currency::deposit(0, 66);
+	pub const MaxPending: u16 = 32;
+}
+
+/// The type used to represent the kinds of proxying allowed.
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, Debug)]
+pub enum ProxyType {
+	Any,
```

Comment:
```suggestion
	/// All calls can be proxied. This is the trivial filter.
	Any,
```

Reply:
We should have at least a comment for each of these. And maybe an example of a case where it would be useful. These docs go a long way to help the devrel team downstream.